### PR TITLE
Kevinloritsch/appDirectory

### PR DESCRIPTION
**Pull request by Kevinloritsch**
image

Added new directories and new pages to set up the basic premises of the website.

### CODE REVIEW

File: src/app/ourTeam/page.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import OurTeam from "@/components/OurTeam/OurTeam";
+
+const OurTeamPage = () => {
```
Comment (SafeDuck) [inline src/app/ourTeam/page.js:3]: rename the folder to `team`

File: src/components/OurTeam/OurTeam.jsx [hunk 2]
```diff
@@ -0,0 +1,5 @@
+const OurTeam = () => {
+  return <div>Our Team</div>;
```
Comment (SafeDuck) [inline src/components/OurTeam/OurTeam.jsx:2]: place this file in the board folder instead, make sure to git pull

File: src/components/events/Events.jsx [hunk 3]
```diff
@@ -0,0 +1,5 @@
+const Events = () => {
+  return <div>Events</div>;
```
Comment (SafeDuck) [inline src/components/events/Events.jsx:2]: place this in the calendar folder in components, make sure to pull

**Review by SafeDuck — CHANGES_REQUESTED** [review]

File: src/components/OurTeam/OurTeam.jsx (hunk 2 above)
Comment (Kevinloritsch) [reply src/components/OurTeam/OurTeam.jsx:2]: Have these folders been created? I pulled but cannot seem to find them.

**Pull request closed without merging**